contracts/VeJoeToken.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title Vote Escrow Joe Token - veJOE
+/// @author Trader Joe
+/// @notice Infinite supply, used to receive extra farming yields and voting power
+contract VeJoeToken is ERC20("VeJoeToken", "veJOE"), Ownable {
```

Comment:
Should this not be a `VeERC20` rather than `ERC20`?

Reply:
ah great catch! will fix